changelog/issue-7128-1.md

```diff
@@ -0,0 +1,6 @@
+audience: users
+level: patch
+reference: issue 7128
+---
+
+non-privileged d2g payloads now map uids and gids from the host into the container, which avoids most issues with permissions when mounting volumes into a container
```

Comment:
See https://github.com/taskcluster/taskcluster/blob/main/dev-docs/best-practices/changelog.md#snippet-text for general guidance when writing change logs.

The change log should be comprised of sentences (i.e. starting with a capital letter, ending with a period). Bug fixes should ideally describe under which circumstances the bug occurred. Something like:

> Generic Worker / D2G bug fix: support has been improved for running Docker Worker tasks with caches under Generic Worker. Previously, caches from a Docker Worker task running under Generic Worker containing files owned by a user other than root would not be owned by the same (container) user when the cache was mounted in a future task. D2G now consistently maps container uids and gids to host subuids and subgids (when caches are used) in order that cache file ownership, as seen from inside the container, is maintained across task runs.

My version might be a bit too wordy, but you get the idea. I really wanted to capture the fix pertains to tasks running under Generic Worker, when a Docker Worker task is submitted, and specifically when caches are used, in order to guarantee consistent file ownership as seen from inside the container.